pkg/scheduler/core/util.go

```diff
@@ -120,3 +120,18 @@ func resortClusterList(clusterAvailableReplicas []workv1alpha2.TargetCluster, sc
 	klog.V(4).Infof("Resorted target cluster: %v", clusterAvailableReplicas)
 	return clusterAvailableReplicas
 }
+
+//attachZeroReplicasCluster  attach cluster in clusters into targetCluster
+//The purpose is to avoid workload not appeared in rb's spec.clusters field
+func attachZeroReplicasCluster(clusters []*clusterv1alpha1.Cluster, targetClusters []workv1alpha2.TargetCluster) []workv1alpha2.TargetCluster {
```

Comment:
First new a `[]workv1alpha2.TargetCluster` with 0 replicas by `clusters`.
Then `util.MergeTargetClusters` to merge.
It could be more concise.